Sources/SwiftDocC/Benchmark/Benchmark.swift

```diff
@@ -133,15 +133,15 @@ public func benchmark<E>(end metric: @autoclosure () -> E?, benchmarkLog log: Be
 ///   - log: The log to add the metric to.
 ///   - body: The closure around which to measure the metric.
 /// - Returns: The return value from the closure.
-public func benchmark<E, Result>(wrap metric: @autoclosure () -> E, benchmarkLog log: Benchmark = .main, body: () throws -> Result) rethrows -> Result where E: BenchmarkBlockMetric {
-    if log.shouldLogMetricType(E.self) {
-        let event = metric()
-        event.begin()
-        let result = try body()
-        event.end()
-        log.metrics.append(event)
-        return result
-    } else {
+public func benchmark<E, Result>(wrap metric: @autoclosure () -> E, benchmarkLog log: Benchmark = .main, body: () throws -> Result) rethrows -> Result
+where E: BenchmarkBlockMetric {
```

Comment:
Just wrapping line length here? Or do we always move the `where` clause to a separate line?

Reply:
My guess is that its's the line length. 

That said considering the two closure parameters, this is probably worth formatting each parameter on a separate line for readability

Reply:
Again worth noting that it won't remove line breaks if they don't conflict with another rule.